core/modules/block/block.admin.inc

```diff
@@ -137,6 +137,14 @@ function block_admin_configure($form, &$form_state, $delta = NULL) {
     '#type' => 'submit',
     '#value' => t('Save block'),
   );
+  $form['actions']['delete'] = array(
+      '#type' => 'link',
+      '#title' => t('Delete'),
+      '#href' => 'admin/structure/block/manage/' . $delta . '/delete',
+      '#options' => array(
+        'attributes' => array('class' => array('button', 'button-secondary', 'form-delete'))
+      ),
```

Comment:
I think we can get rid of the entire `#options` array here, which actually isn't valid in the first place (it would be `#attributes`). But all the necessary classes are added automatically already for you.

And per @stpaultim's feedback in https://github.com/backdrop/backdrop-issues/issues/3299#issuecomment-485192642, we should separately added a `#access` property here that only shows the button if the block is not new.

Reply:
I just used $delta to do this, is that okay @quicksketch? It still seems to work...

Reply:
@quicksketch I was curious about how it's done in the node edit form, but that one does use `#options` and `#attributes`. And it also wraps it in a check for `nid`. So maybe this is good enough for core? Or should we be improving the node edit form too?

```
  if (!empty($node->nid) && node_access('delete', $node)) {
    // Use a link for the delete button so the form doesn't need to validate.
    $form['actions']['delete'] = array(
      '#type' => 'link',
      '#title' => t('Delete'),
      '#href' => "node/{$node->nid}/delete",
      '#options' => array(
        'query' => isset($_GET['destination']) ? backdrop_get_destination() : array(),
        'attributes' => array('class' => array('button', 'button-secondary', 'form-delete'))
      ),
      '#weight' => 15,
    );
  }
```

Reply:
Looking at this again I realized it's `#type` is `link`, I was thinking this was a button. Now this all makes sense. theme_link() has an `options` parameter, so that's what's being passed in here. Seems like we might want to consider making `attributes` a parameter so that you can use `#attributes` instead... but that's out of scope. This code is fine and replicates what we have already.